templates/frontend/pages/catalog.tpl

```diff
@@ -24,6 +25,22 @@
 		{translate key="catalog.browseTitles" numTitles=$total}
 	</div>
 
+	{* Series List *}
+	{if $activeTheme->getOption('showCatalogueSeriesListing') && $contextSeries|@count > 1}
+		<nav class="pkp_series_nav_menu" role="navigation" aria-label="Series">
+			<strong>{translate key="series.series"}:</strong>
```

Comment:
This needs to be `h2` for accessibility, but styled like it is just bold.

Reply:
check